src/interactor.js

```diff
@@ -180,6 +184,9 @@ export default class Interactor extends Convergence {
     return $$(selector, this.$root);
   }
 
+  chain(parent) {
```

Comment:
What's this for?

Reply:
Left over from trying to figure out the best way to do this parent stuff. ✂️